jinahub/indexers/searcher/compound/FaissPostgresSearcher/faisspsql.py

```diff
@@ -2,50 +2,172 @@
 __license__ = "Apache-2.0"
 
 import copy
-from typing import Dict
+import datetime
+import functools
+from typing import Dict, Optional
 
-from jina import requests, DocumentArray, Executor
-from jina.logging.logger import JinaLogger
+from jina import DocumentArray, Executor, requests
+from jina_commons import get_logger
 
 try:
     from jinahub.indexers.searcher.FaissSearcher import FaissSearcher
-except:
-    from jina_executors.indexers.searcher.FaissSearcher.faiss_searcher import FaissSearcher
+except ImportError:  # noqa: E722
+    from jina_executors.indexers.searcher.FaissSearcher.faiss_searcher import (
+        FaissSearcher,
+    )
```

Comment:
This whole thing, plus I guess the installations around it, can now be removed because jina does not use `jinahub` anymore, so there will be no more conflicts.

Reply:
![image](https://user-images.githubusercontent.com/8330330/131804682-1562a4d0-1932-43dd-adc2-ffba8ab75090.png)